### ISSUE TITLE
support sui not only shizuku

### ISSUE THREAD

**Issue by yennor** [ThePBone/RootlessJamesDSP#177]
I've got sui installed which also provides the shizuku API.

When I click the button "Grant access" in rootlessJamesDSP it tells me that shizuku is not installed.
According to sui, if the application supports Shizuku API v11+ it should also work with sui (if not intentionally broken).
As it seems you are using a new API: shizukuVersion = "12.2.0"
So I guess there most be something else wrong?
I've tried out if sui is correctly installed with ashell. Ashell discovers it and works perfectly. So my sui installation should be ok.

**Comment by CoelacanthusHex** [comment]
> shizuku

Yeah, it seems here: https://github.com/ThePBone/RootlessJamesDSP/blob/c31cddf28de2e31c2a9017ae70534b5032ba1042/app/src/main/java/me/timschneeberger/rootlessjamesdsp/fragment/OnboardingFragment.kt#L223

**Comment by chronicsilver23** [comment]
If you install Shizuku app but don't activate it Sui picks up the app request.  I was wondering if it would  continue to require Shizuku,, but after uninstalling it it's still using Sui API.

OnePlus 7 Pro
Android 10
Zygist-Sui 13.5.1
